src/TopoMojo.Hypervisor/Proxmox/ProxmoxClient.cs

```diff
@@ -689,6 +714,12 @@ private Vm LoadVm(IClusterResourceVm pveVm)
                 Tags = pveVm.Tags
             };
 
+            if (_tasks.ContainsKey(vm.Id))
```

Comment:
Newer versions of .NET ask you to do this via `.TryGetValue(key, out var whatever)`. Not a huge deal either way, I suspect.

Reply:
Yea, I was just copying the vmware implementation. Will go back and optimize things in a later PR.